automated/linux/kselftest/parse-output.py

```diff
@@ -74,9 +80,19 @@ def make_names_unique(results):
             r["name"] += f"_dup{namecounts[name]}"
 
 
+def make_log_files(results):
+    for r in results:
+        name = r["name"]
+        if r["result"] == "fail":
+            log_file = open(f"output/{name}.log", "w")
```

Comment:
it would be a good idea to catch any IO exceptions here. IIRC this script is run on DUT and you may run out of disk space.

Reply:
Thanks for the comment, I added an exception block. Let me know if there are any issues.

Reply:
should be OK now.